app/src/main/java/com/android/vending/billing/utils/IabHelper.java

```diff
@@ -536,7 +536,7 @@ public Inventory queryInventory(boolean querySkuDetails, List<String> moreSkus)
     /**
```

Comment:
This is a third party library, prefer not to make changes here, as it makes it difficult to merge in future changes to this library.